content/applications/finance/fiscal_localizations/uruguay.rst

```diff
@@ -0,0 +1,362 @@
+- :guilabel:`Due date`: to compute if the invoice is due now or later (contado or crédito
+  respectively).
+- :guilabel:`Journal`: select the electronic sales journal.
+- :guilabel:`Document Type`: document type in this format, for example, `(111) e-Invoice`.
+- :guilabel:`Products`: specify the product with the correct taxes.
+
+.. note::
+   Every document type has a specific credit note and debit note (e.g., the document type *(111)
+   e-Invoice* has an *(112) e-Invoice Credit Note*).
+
+Customer credit note
+~~~~~~~~~~~~~~~~~~~~
+
+The :doc:`Customer credit note <../accounting/customer_invoices/credit_notes>` is an electronic
+document that, when validated, is sent to |DGI| via Uruware. It is necessary to have a validated
+(posted) invoice to register a credit note. On the invoice there is a button named :guilabel:`Credit
+note`, click this button to be directed to the :guilabel:`Create credit note` form, then complete
+the following information:
+
+- :guilabel:`Reason`: type the reason for the credit note.
+- :guilabel:`Journal`: select the journal that has to be electronic and has the :guilabel:`Use
+  Documents?` option active.
+- :guilabel:`Document Type`: select the credit note document type.
+- :guilabel:`Reversal Date`: type the date.
+
+Customer debit note
+~~~~~~~~~~~~~~~~~~~
+
+The :doc:`Customer debit note <../accounting/customer_invoices/credit_notes>` is an electronic
+document that, when validated, is sent to |DGI| via Uruware. It is necessary to have a validated
+(posted) invoice to register a debit note. On the action within an invoice there is a option named
+:guilabel:`Debit note`, click on this action to be directed to the :guilabel:`Create credit note`
+form, then complete the following information:
+
+- :guilabel:`Reason`: type the reason for the debit note.
+- :guilabel:`Journal`: select the journal that has to be electronic and has the :guilabel:`Use
+  Documents?` option active.
+- :guilabel:`Copy lines`: mark checkbox to copy the invoice lines to the debit note.
+- :guilabel:`Debit note date`: type the date.
+
+.. note::
+   Confirm the invoice and it will be created with an internal reference. To send the document to
+   |DGI| via Uruware, click on :guilabel:`Send and Print` and select the checkbox :guilabel:`Create
+   CFE`. The legal document sequence (number) is brought from Uruware once the document has been
+   processed. Make sure you have |CAEs| available in Uruware.
+
+.. note::
+   The PDF of the validated document is pulled from Uruware following the specification by the
+   Uruguayan government (DGI).
+
+Addendas and disclosures
+========================
+
+*Addendas* and *disclosures* are additional notes and comments added to an electronic document that
+can be mandatory or optional. To create a new addenda, go to :menuselection:`Accounting --> Settings
+--> DGI --> Addendas and disclosures`.
+
+To create an addenda, the following information is required:
```

Comment:
```suggestion
Enter the the following information:
```
so we don't repeat *To create an addenda*

Reply:
Just remove one of the "the"s 😅, thanks @samueljlieber

Reply:
No worries! Thanks a lot! 😉